docs/included.md

```diff
@@ -13,6 +13,10 @@
   - [API](api.md#scores.continuous.additive_bias)
   - [Tutorial](project:./tutorials/Additive_and_multiplicative_bias.md)
   - [Mean Error (WWRP/WGNE Joint Working Group on Forecast Verification Research)](https://www.cawcr.gov.au/projects/verification/#meanerror)
+* - Additive Bias percentage
+  - [API](api.md#scores.continuous.additive_bias_percentage)
+  - [Tutorial](project:./tutorials/Additive_and_multiplicative_bias.md)
+  - [https://search.r-project.org/CRAN/refmans/hydroGOF/html/pbias.html)  
```

Comment:
```suggestion
  - [Percent Bias (CRAN hydroGOF)](https://search.r-project.org/CRAN/refmans/hydroGOF/html/pbias.html)  
```

Reply:
1. For consistency with other entries, I think it is worth adding some of kind of description that hyperlinks to the URL. I wasn't 100% sure what a good descriptor would be, so picked "Percent Bias (CRAN hydroGOF)". Please don't feel obliged to use that exactly - if you have a better descriptor, please include that instead!
2. Is there an academic paper(s) that describes/defines this metric? If so, it would be great if we could list that either instead of or in addition to the R link above.

Reply:
After some more background reading, this score is clearly known universally as "Percent bias", so I suggest we just call "Percent Bias (PBIAS)" that (rather than additive percentage bias), despite my earlier thoughts on the matter.

Reply:
@Steph-Chong , here are some of the academic papers: 

1. Sorooshian, S., Duan, Q., & Gupta, V. K. (1993). Calibration of rainfall-runoff models: Application of global optimization to the Sacramento Soil Moisture Accounting Model. Water Resources Research, 29(4), 1185-1194. doi:https://doi.org/10.1029/92WR02617
2. Alfieri, L., Pappenberger, F., Wetterhall, F., Haiden, T., Richardson, D., & Salamon, P. (2014). Evaluation of ensemble streamflow predictions in Europe. Journal of Hydrology, 517, 913-922. doi:http://dx.doi.org/10.1016/j.jhydrol.2014.06.035
3. Dawson, C. W., Abrahart, R. J., & See, L. M. (2007). HydroTest: A web-based toolbox of evaluation metrics for the standardised assessment of hydrological forecasts. Environmental Modelling and Software, 22(7), 1034-1052.

Reply:
@durgals Here are three more links to papers covering PBIAS which I think may be more useful introductions to the score. However, as I'm not a Hydrologist, if you could take a look, that would be great. Ideally, one of the links from included.md should go to a non-paywalled article. I'll also leave a comment about setting out references in the docstring.

https://doi.org/10.3133/sir20135111
https://doi.org/10.13031/2013.23153
https://doi.org/10.13031/trans.58.10715

Reply:
Added one of the citations which is more relevant here. Two of the citations are very similar.